specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2021-04-01-preview/apimschema.json

```diff
@@ -0,0 +1,329 @@
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Lists a collection of Schema entities.",
+            "schema": {
+              "$ref": "./definitions.json#/definitions/SchemaCollection"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./apimanagement.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-odata": "./definitions.json#/definitions/SchemaContract"
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/schemas/{schemaId}": {
+      "head": {
+        "tags": [
+          "Schema"
+        ],
+        "operationId": "Schema_GetEntityTag",
+        "description": "Gets the entity state (Etag) version of the Schema specified by its identifier.",
+        "x-ms-examples": {
+          "ApiManagementHeadApi": {
+            "$ref": "./examples/ApiManagementHeadSchema.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "./apimanagement.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ServiceNameParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/SchemaIdParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Specified Schema entity exists and current entity state version is present in the ETag header.",
+            "headers": {
+              "ETag": {
+                "description": "Current entity state version. Should be treated as opaque and used to make conditional HTTP requests.",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./apimanagement.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "Schema"
+        ],
+        "operationId": "Schema_Get",
+        "description": "Gets the details of the Schema specified by its identifier.",
+        "x-ms-examples": {
+          "ApiManagementGetSchema1": {
+            "$ref": "./examples/ApiManagementGetSchema1.json"
+          },
+          "ApiManagementGetSchema2": {
+            "$ref": "./examples/ApiManagementGetSchema2.json"
+          }
+        },
+        "produces": [
```

Comment:
It's not going to produce anything but application/json. Schema type will be conveyed in schemaType property.